ecl/network/v2/load_balancers/requests.go

```diff
@@ -0,0 +1,183 @@
+package load_balancers
+
+import (
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/pagination"
+)
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the Load Balancer attributes you want to see returned. SortKey allows you to sort
+// by a particular Load Balancer attribute. SortDir sets the direction, and is either
+// `asc' or `desc'. Marker and Limit are used for pagination.
+type ListOpts struct {
+	AdminUsername      string `q:"admin_username"`
+	AvailabilityZone   string `q:"availability_zone"`
+	DefaultGateway     string `q:"default_gateway"`
+	Description        string `q:"description"`
+	ID                 string `q:"id"`
+	LoadBalancerPlanId string `q:"load_balancer_plan_id"`
```

Comment:
This filed should be changed for consistency.
```suggestion
	LoadBalancerPlanID string `q:"load_balancer_plan_id"`
```

Reply:
Thank you for suggestion!